.github/workflows/lint.yml

```diff
@@ -0,0 +1,19 @@
+name: Lint
+
+on: [push]
+
+jobs:
+  eslint:
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
please separate lint and typescript so that they are 2 steps. Thanks to that it's easier to investigate what's wrong

Reply:
@Okelm done